k8s.io/configmap-nginx.yaml

```
@@ -232,6 +232,7 @@ data:
         location / {
           rewrite ^/api-review$      https://github.com/kubernetes/community/blob/master/sig-architecture/api-review-process.md redirect;
           rewrite ^/bot-commands$    https://prow.k8s.io/command-help redirect;
+          rewrite ^/calendar$        https://calendar.google.com/calendar/embed?src=calendar%40kubernetes.io redirect;
```

```suggestion
          rewrite ^/calendar$        https://www.k8s.dev/calendar
```
no objections to adding this with the suggestion

(also I haven't actually deployed changes to this, so I would rather defer to someone else who has for the /approve)

---

ping @dims to accept this suggestion